core/cat/looking_glass/LayerTwo.py

```diff
@@ -0,0 +1,43 @@
+import openai
+import re
+
+class Layer2LLM:
+    def __init__(self, initial_prompt):
+        openai.api_key = "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"
```

Comment:
@0xSorcerer watch out you shared a key on. You'd better delete it and create a new one

Reply:
OpenAI automatically invalidates API keys shared on GitHub. Thanks for the heads-up though!